models/articles.models.js

```diff
@@ -4,6 +4,9 @@ function fetchArticleById(id) {
   return db
     .query("SELECT * FROM articles WHERE article_id = $1", [id])
     .then((article) => {
+      if (article.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article does not exist" });
+      }
       return article;
```

Comment:
return article.rows here so you don't need to access the property in the controller.